src/receive_message.rs

```diff
@@ -82,8 +82,8 @@ impl ReceiveMessage {
     /// Retrieves the length of the received message.
     ///
     /// The length represents the length of the payload.
-    pub fn length(&self) -> u32 {
-        self.inner.length
+    pub fn length(&self) -> u64 {
```

Comment:
might do fancier stuff later, for now I just wanted to update to the latest client with the min amount of changes.